dali/pipeline/operators/reader/video_reader_op_test.cc

```diff
@@ -68,4 +71,60 @@ TEST_F(VideoReaderTest, ConstantFrameRate) {
   ASSERT_EQ(frames_shape[0][0], sequence_length);
 }
 
+TEST_F(VideoReaderTest, MultipleVideoResolution) {
```

Comment:
Please check `CalcOpticalFlowTest` to see how to skip test in GTest. We need to that if driver version is insufficient to support multi-resolution videos.